kernel/kernel-api/src/main/java/io/delta/kernel/Table.java

```diff
@@ -66,4 +66,15 @@ Snapshot getLatestSnapshot(TableClient tableClient)
      */
     Snapshot getSnapshotAtVersion(TableClient tableClient, long versionId)
         throws TableNotFoundException;
+
+    /**
+     * Get the snapshot at the given {@code timestamp}.
```

Comment:
Can you be a bit more specific?

```
- timestamp T-1: Table Version 10 created
- timestamp T: 
- timestamp T+1: Table Version 11 created
```

if we query for the snapshot at version T, what do we return?

Reply:
Updated the docs to be more clear